tools/dependencies-report/src/main/resources/notices/jruby-complete-9.1.13.0-NOTICE.txt

```diff
@@ -0,0 +1,279 @@
+JRuby is Copyright (c) 2007-2018 The JRuby project
```

Comment:
the jruby project's COPYING contains additional copyright notices that we are obligated to propagate; in a conversation via e-mail with Legal, we decided to include the notices call-out, followed by the full-text of the EPL-1.0 (copied from https://github.com/jruby/jruby/blob/9.1.13.0/COPYING#L1-L340).

See: https://github.com/elastic/logstash/pull/9549/files#diff-eedb8d48d5da4dd9d4e2fca4b66d6eea